src/test/java/it/eng/idsa/businesslogic/service/impl/SelfDescriptionServiceImplTest.java

```diff
@@ -50,23 +47,23 @@
 	@BeforeEach
 	public void setup() throws ConstraintViolationException, URISyntaxException {
 		MockitoAnnotations.initMocks(this);
-		when(dapsService.getJwtToken()).thenReturn("mockTokenValue");
 		when(configuration.getInformationModelVersion()).thenReturn(infoModelVersion);
 		when(configuration.getConnectorURI()).thenReturn(URI.create(connectorURI));
 		when(configuration.getTitle()).thenReturn(title);
 		when(configuration.getDescription()).thenReturn(description);
 		when(configuration.getCurator()).thenReturn(URI.create(curratorURI));
+		when(configuration.getDefaultEndpoint()).thenReturn(URI.create("https://defaultEndpoint"));
 		when(configuration.getMaintainer()).thenReturn(URI.create(maintainerURI));
 
-		selfDefinitionService = new SelfDescriptionServiceImpl(dapsService, configuration, dataAppService);
+		selfDefinitionService = new SelfDescriptionServiceImpl( configuration, dataAppService);
 		selfDefinitionService.initConnector();
 	}
 
 	@Test
 	public void getConnectionString() throws IOException {
 		String connectionString = selfDefinitionService.getConnectorAsString();
```

Comment:
Can you rename this variable - connectionString  to selfDescription?
Describes better what variable holds